pkg/minikube/machine/cache_images.go

```diff
@@ -652,6 +653,7 @@ func RemoveImages(images []string, profile *config.Profile) error {
 			if err != nil {
 				failed = append(failed, m)
 				klog.Warningf("Failed to remove images for profile %s %v", pName, err.Error())
+				out.WarningT("Failed to remove images for profile {{.pNmae}} {{.error}}", out.V{"pNmae": pName, "error": err.Error()})
```

Comment:
Typo `pNmae`

Reply:
sorry. I havs fixed it. Thanks very much. @spowelljr